tests/Functional/Resources/Events/_fixtures/response/list_default.json

```diff
@@ -86,6 +86,138 @@
         "import_meta": null
       }
     },
+    {
```

Comment:
Should we add some assertions for the new properties, e.g. `unit_price_overrides`?